assignment-7/src/components/users-component/index.js

```diff
@@ -1,4 +1,75 @@
+import { useParams } from "react-router";
+import { useEffect, useState } from "react";
+import fetchapirequest from "../../utils/utils";
+import "./index.scss";
+
 const UserComponent = () => {
-  return <div>users data</div>;
+  let name = "";
+  const [userdetail, setuserdetail] = useState({});
+  const { id } = useParams();
+  useEffect(() => {
+    fetchapirequest([id], "https://api.github.com/users")
+      .then((data) => {
+        console.log(data);
```

Comment:
Can you tell why [id] is used here? I didn't get it

Reply:
The fetchapirequest function accepts an array as its first argument because I used map on it.
If I wont wrap it inside [], it will give me an error.

I'm also using the same function to get all user information as well as user-specific information.